src/ansys_sphinx_theme/theme/ansys_sphinx_theme/components/search-button-field.html

```diff
@@ -0,0 +1,14 @@
+
+<div class="search-button" id="search-button">
```

Comment:
What about renaming this to? Later, we can use `search-button` and `search-input` to other elements.

```suggestion
<div class="search-bar" id="search-bar">
```